src/main/resources/view/MainWindow.fxml

```diff
@@ -14,7 +14,7 @@
 <fx:root type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1"
          title="Address App" minWidth="450" minHeight="600" onCloseRequest="#handleExit">
```

Comment:
The title of the app is still Address App. Consider changing to our app name "Agent Assist"!

Reply:
Will be changed by my separate PR, see #83